frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -83,6 +83,14 @@ export default class DashCard extends Component {
         const isSlow = loading && _.some(series, (s) => s.duration) && (usuallyFast ? "usually-fast" : "usually-slow");
 
         const hasUnmappedParameters = _.any(series, (s) => s.json_query && _.any(s.json_query.parameters, (p) => p.target == null));
```

Comment:
@tlrobinson I'm trying to understand if the `hasUnmappedParameters` flag is still relevant. Does it have the same purpose as the `isMappedToAllParameters` flag I added below for this issue, just not working as intended? Or does it have some different purpose altogether?